### PR TITLE
ENH: Highlight downstream analysis outside of Q2

### PR DESCRIPTION
**Pull request by adamovanja**
This PR links the tutorial sections on downstream analysis within and outside of Q2 in the ReadMe file. The respective sections were rewritten and the tutorial was restructured to have fewer high-level headings.

### CODE REVIEW

**Review by LenaFloerl — APPROVED** [review]
Thanks for the update - sounds great!

File: README.md [hunk 1]
```diff
@@ -187,6 +189,11 @@ where:
 - `--p-email` is your email address (required by NCBI)
 - `--output-dir` directory where the downloaded metadata, sequences and IDs for failed downloads are stored as QIIME 2 artifacts
 
+## Downstream analysis
+For more information on how to use q2-fondue outputs within the QIIME 2 ecosystem see section [Downstream analysis in QIIME 2](./tutorial/tutorial.md#downstream-analysis-in-qiime-2) in the tutorial.     
+
+To learn more on how to prepare q2-fondue outputs for further analysis outside of QIIME 2 see tutorial section [Prepare downstream analysis outside of QIIME 2](./tutorial/tutorial.md#prepare-downstream-analysis-outside-of-qiime-2).
```
Comment (nbokulich) [inline README.md:195]: This section/sentence should perhaps be preceded with an acknowledgement that... "Some downstream analyses may need to rely on tools outside of QIIME 2. q2-fondue can be used for diverse applications, and data can be exported directly into FASTQ and other interoperable formats. Note that the exported files will no longer contain integrated provenance information (which is unique to QIIME 2 Artifacts), but this metadata can be exported also and the original artifacts will retain the provenance data for traceability purposes."

**Review by nbokulich — CHANGES_REQUESTED** [review]
thanks @adamovanja ! Some minor changes to propose.

File: tutorial/tutorial.md [hunk 2]
```diff
@@ -282,22 +282,21 @@ qiime tools view fondue-output/dada2_table.qzv
...
 qiime tools extract \
       --input-path metadata.qza \
       --output-path metadata
 ```
 This creates a metadata directory with all information on provenance tracking, 
-and in the folder *data* we find the *sra-metadata.tsv*.
+and in the folder *data* we find the *sra-metadata.tsv* containing all metadata of the initially requested accession IDs.
 
-#### Get FASTA files 
+### Extract the retrieved sequences as FASTA/FASTQ files
```
[the remark this answers is not in the document]
Comment (adamovanja) [reply tutorial/tutorial.md:299]: you are right. It seems that the default is FASTQ, which is also what we support in q2-fondue

File: README.md (hunk 1 above)
Comment (adamovanja) [reply README.md:198]: I like this. Your suggestion is included with a slight edit in the second sentence.

File: README.md [hunk 3]
```diff
@@ -187,6 +189,11 @@ where:
 - `--p-email` is your email address (required by NCBI)
 - `--output-dir` directory where the downloaded metadata, sequences and IDs for failed downloads are stored as QIIME 2 artifacts
 
+## Downstream analysis
```
[the remark this answers is not in the document]
Comment (adamovanja) [reply README.md]: How about two separate headings: "Downstream analysis in QIIME 2" & "Exporting data for downstream analyses outside of QIIME 2" (included in the newest edits)?

**Comment by adamovanja** [comment]
thanks for the reviews @LenaFloerl & @nbokulich 🎉 . 

@nbokulich I have addressed your comments in the latest commit. Let me know what you think.

**Review by nbokulich — APPROVED** [review]
LGTM thanks @adamovanja !

**Pull request merged**